src/main/java/pl/psnc/dei/iiif/ConversionDataTransformer.java

```diff
@@ -0,0 +1,53 @@
+package pl.psnc.dei.iiif;
+
+import org.apache.jena.atlas.json.JSON;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import pl.psnc.dei.model.conversion.ConversionData;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.stream.Collectors;
+
+public class ConversionDataTransformer {
```

Comment:
Utility classes, which has only static methods, should have private constructor.

Reply:
added